### PR TITLE
Uppercase count

### PR DESCRIPTION
**Pull request by hyades**
Closes https://github.com/BlakeGuilloud/ganon/issues/193

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: lib/countUpperCase.js [hunk 1]
```diff
@@ -4,8 +4,7 @@
  */
 
 function countUpperCase(str) {
-  // Your code goes here.
-  return;
+  return str.replace(/[^A-Z]/g, "").length;
```
Comment (ktilcu) [inline lib/countUpperCase.js]: I have a tip for the future. If we are looking for capital letters to count then the easiest implementation to understand is one that does the same thing. I think of it as a direct solution. This one is sort of indirect in that it finds and removes everything that we aren't looking for.

Just a thought and a tip.

**Comment by hyades** [comment]
@ktilcu please check now.

**Comment by BlakeGuilloud** [comment]
Hey @hyades ! If you don't mind pulling in the latest master, we can get this merged in! thanks!

**Comment by ktilcu** [comment]
Oh I see. Someone else already made this `countLowerCase` skeleton

**Comment by hyades** [comment]
@ktilcu @BlakeGuilloud Haha. Okay added another one :)

**Pull request merged**